sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -759,22 +769,9 @@ def waiting_time_compensation_with_sleep(time_start, time_to_wait):
 # Update port SFP status table on receiving SFP change event
 
 
-def update_port_transceiver_status_table(logical_port_name, status_tbl, status, has_error=False):
-    if not has_error:
-        fvs = swsscommon.FieldValuePairs([('status', status), ('error', 'N/A')])
-        status_tbl.set(logical_port_name, fvs)
-    else:
-        error_list = []
-        int_status = int(status)
-        for error_code, error_msg in sfp_status_helper.SFP_STATUS_ERR_DICT.items():
-            if error_code & int_status:
-                error_list.append(error_msg)
-        if error_list:
-            fvs = swsscommon.FieldValuePairs([('status', str(int_status & 1)), ('error', '|'.join(error_list))])
-            status_tbl.set(logical_port_name, fvs)
-        else:
-            # SFP return unkown event, just ignore for now.
-            helper_logger.log_warning("Got unknown event {}, ignored".format(status))
+def update_port_transceiver_status_table(logical_port_name, status_tbl, status, error_descriptions='N/A'):
```

Comment:
It seems error_descriptions will never be 'N/A'

Reply:
In a normal [insertion](https://github.com/Junchao-Mellanox/sonic-platform-daemons/pull/26/files/5a5aac4bbb6b7baba934365dcfc080b218f2b749#diff-fc70c43977a1ce57f3a6e9d7d09cb7ea5204f52b3adf3162936205e5680efb2fR1066)/[removal](https://github.com/Junchao-Mellanox/sonic-platform-daemons/pull/26/files/5a5aac4bbb6b7baba934365dcfc080b218f2b749#diff-fc70c43977a1ce57f3a6e9d7d09cb7ea5204f52b3adf3162936205e5680efb2fR1081) flow, the `error_descriptions` won't be fed therefore being `N/A`.